lib/types/pact_value.ex

```diff
@@ -17,7 +17,8 @@ defmodule Kadena.Types.PactValue do
           | String.t()
           | PactInt.t()
           | PactDecimal.t()
-          | PactValuesList.t()
+          | PactValue.t()
+          | list(literal())
```

Comment:
@EdwinGuayacan we cannot define `list(literal())` if type **literal** does not exists before

Reply:
no :(